orchestration/src/test/java/com/sap/ai/sdk/orchestration/OrchestrationUnitTest.java

```diff
@@ -85,6 +85,35 @@ void testCompletion() {
     assertThat(result.getContent()).isNotEmpty();
   }
 
+  @Test
+  void testGrounding() {
+    stubFor(
+        post(anyUrl())
+            .willReturn(
+                aResponse()
+                    .withBodyFile("groundingResponse.json")
+                    .withHeader("Content-Type", "application/json")));
+    final var response = client.chatCompletion(prompt, config);
+    final var result = response.getOriginalResponse();
+    var llmChoice =
+        ((LLMModuleResultSynchronous) result.getOrchestrationResult()).getChoices().get(0);
+
+    assertThat(result.getModuleResults().getGrounding().getData().toString())
+        .isEqualTo(
+            "{grounding_query=grounding call, grounding_result=Joule is the AI copilot that truly understands your business. Joule revolutionizes how you interact with your SAP business systems, making every touchpoint count and every task simpler.```It enables the companion of the Intelligent Enterprise, guiding you through content discovery within SAP Ecosystem, and giving a transparent role-based access to the relevant processes from everywhere. This is the one assistant experience, a unified and delightful user experience across SAP’s Ǯ solution portfolio.}");
```

Comment:
(personal preference)

Personally, I feel this is a bit long for a unit test

Reply:
Exchanged `isEqualTo` to `startsWith` and shortened the expected strings.